src/components/profile/ViewAvatar.js

```diff
@@ -1,11 +1,27 @@
 // @flow
 import React from 'react'
+import normalize from 'react-native-elements/src/helpers/normalizeText'
+import Icon from 'react-native-elements/src/icons/Icon'
+import { StyleSheet, View } from 'react-native'
 import GDStore from '../../lib/undux/GDStore'
-import { UserAvatar } from '../common'
+import { Section, UserAvatar, Wrapper } from '../common'
 import CameraButton from './CameraButton'
 
 const TITLE = 'View Avatar'
 
+const CloseButton = ({ handlePress, containerStyles = {}, iconStyles = {} }) => (
+  <View style={[styles.container, containerStyles]}>
+    <Icon
+      onPress={handlePress}
+      size={normalize(20)}
+      color="#0C263D"
```

Comment:
can we wrap `CloseButton` with `withStyles` and use `theme.colors.darkBlue`?

Reply:
Done